jenkins-config.yaml

```diff
@@ -0,0 +1,11 @@
+unclassified:
+  globalDatabaseConfiguration:
+    database:
+      postgreSQL:
+        database: postgres
+        hostname: db
+        password: postgres
+        username: postgres
+        validationQuery: "SELECT 1"
+  junitTestResultStorage:
+    storage: "database"
```

Comment:
```suggestion
    storage: "database"

```

Reply:
I've added a line break to the end of the file.